core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -180,7 +180,9 @@ DistributedTransaction begin(String txId, Isolation isolation, SerializableStrat
         new Snapshot(txId, isolation, strategy, tableMetadataManager, parallelExecutor);
     CrudHandler crud =
         new CrudHandler(storage, snapshot, tableMetadataManager, isIncludeMetadataEnabled);
-    ConsensusCommit consensus = new ConsensusCommit(crud, commit, recovery);
+    ConsensusCommitMutationOperationChecker operationChecker =
+        new ConsensusCommitMutationOperationChecker(tableMetadataManager);
```

Comment:
Instead of creating a `ConsensusCommitMutationOperationChecker` instance each time, can we hold it as an instance field of `ConsensusCommitManager` and pass it to `ConsensusCommit` instances?

Reply:
That is a good point, thank you. 
My bad, I did it for the 2pc transaction manager but not for this one. 
Revised in https://github.com/scalar-labs/scalardb/pull/873/commits/de06ddcebf8e54237bba0b7cfbc61935d00c8035